threads.js

```diff
@@ -2036,14 +2036,32 @@ Process.prototype.reportProduct = function (a, b) {
     return +a * +b;
 };
 
+function QuotientNaNException(numerator, denominator) {
+    this.name = "Quotient exception";
```

Comment:
Can we avoid using the word "exception"? I think "error" is much more clear to the average human.